dotcom-rendering/src/web/components/MostViewedRightWrapper.importable.tsx

```diff
@@ -11,8 +11,14 @@ type Props = {
 // Minimum height needed to render MostViewedRight is its own outer height.
 const HEIGHT_REQUIRED = 482 + 24 + 24;
 
-const flexGrow = css`
-	flex-grow: 1;
+const MOSTVIEWED_STICKY_HEIGHT = 1059;
+
+// Styling the data island root so it stretches to cover the full height available in the container.
+// Requires us to subtract the height of its sibling in the container (StickyAd).
+const stretchWrapperHeight = css`
+	display: flex;
+	flex-direction: column;
+	height: ${`calc(100% - ${MOSTVIEWED_STICKY_HEIGHT}px)`};
```

Comment:
This css was previously being added by `MostViewedRightIsland` but is now included here